dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseUtils.java

```diff
@@ -1347,6 +1348,19 @@ public void run()
     /**
      * Determine the type of Database, based on the DB connection.
      * 
+     * @return a DB keyword/type (see DatabaseUtils.DBMS_* constants)
+     * @throws SQLException if database error
+     */
+    public static String getDbType()
+            throws SQLException
+    {
+	Connection connection=getDataSource().getConnection();
+        return getDbType(connection);
+    }
```

Comment:
This method should be removed. It's duplicative of the `Context.getDbType()` method: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/core/Context.java#L179